### PR TITLE
release-qual test: add logic to publish monitor status

### PR DESCRIPTION
**Pull request by richardwxn**
Add logic to write monitor status to spanner
1. initial status would be written to spanner
2. the webhook would handle new alerts and update status in spanner

Should work with the change: https://github.com/istio/bots/pull/261

### CODE REVIEW

**Comment by therealmitchconnors** [comment]
Is there a design or architecture doc explaining this change?  I'm not clear on the purpose of the webhook and other http logic...

**Comment by richardwxn** [comment]
@therealmitchconnors yes, https://docs.google.com/document/d/1CycOfaHtvFR8Fi19GPdurUG_NERxBnqm4RetKLwIh7I/edit

File: perf/stability/alertmanager/webhook.go [hunk 1]
```diff
@@ -15,26 +15,141 @@
...
-		return
 	}
-	log.Printf("Alerts: GroupLabels=%v, CommonLabels=%v", data.GroupLabels, data.CommonLabels)
-	//TODO(richardwxn): process alert
+	log.Printf("alerts: GroupLabels=%v, CommonLabels=%v", data.GroupLabels, data.CommonLabels)
+	var monitorList []SingleMonitorStatus
+	var errs error
 	for _, alert := range data.Alerts {
-		log.Printf("Alert: status=%s,Labels=%v,Annotations=%v", alert.Status, alert.Labels, alert.Annotations)
+		ms, err := convertPromAlertToInternalMonitorStatus(alert)
```
Comment (mandarjog) [inline perf/stability/alertmanager/webhook.go:141]: consider adding link and other details to the spanner table.

Comment (mandarjog) [reply perf/stability/alertmanager/webhook.go:141]: In the UI, ensure that the UI is color coded.

If anything needs attention it should be RED :-)

**Comment by mandarjog** [comment]
@therealmitchconnors PTAL.

File: perf/stability/alertmanager/webhook.go (hunk 1 above)
Comment (richardwxn) [reply perf/stability/alertmanager/webhook.go]: yes, added different colors in the dashboard, check it out here: http://eng.istio.io/releasequal

**Review by therealmitchconnors — APPROVED** [review]

**Pull request merged**